components/PlantList.js

```diff
@@ -2,19 +2,21 @@ import styled from "styled-components";
 import PlantCard from "./PlantCard";
 
 export default function PlantList({ plants, toggleFavourite }) {
-  if (!plants || plants.length === 0) return <p>No plants found</p>;
+  if (!plants || plants.length === 0)
+    return (
+      <p>
+        There are no plants here yet! Start adding some to create your own plant
+        collection.
+      </p>
+    );
 
   return (
-    <StyledList>
+    <ul>
```

Comment:
Currently your `<ul>` and `<li>` is not styled. Wrapping the `<ul>` and `<li>` elements in styled components can give you more control over presentation. But this is fine by now and could be changed in the future.